### ISSUE TITLE
false positive fatal unknow rule errors after upgrade

### ISSUE THREAD

**Issue by openmindculture** [stylelint/stylelint-config-standard#316]
### What steps are needed to reproduce the bug?

Use a working configuration that works with "stylelint": "^14.14.0", "stylelint-config-html": "^1.1.0",    "stylelint-config-standard": "^29.0.0", and disable some opinionated rules like
`max-empty-lines`, `number-no-trailing-zeros`, `string-quotes`, `value-list-comma-space-after` by setting them to `null` in `.stylelintrc.json`.

Upgrade stylelint and standard configuration to the latest versions:

```
        "stylelint": "^16.2.0",
        "stylelint-config-html": "^1.1.0",
        "stylelint-config-standard": "^36.0.0"
```

Now the linting breaks because of those rules. This should be non-fatal deprecation warnings instead.

### What did you expect to happen?

Linting still works after an upgrade. Errors about removed rules should be non-fatal deprecation warnings instead.

### What actually happened?

Now the linting breaks because of those rules.

**Comment by ybiquitous** [comment]
@openmindculture That stylistic rules have been removed in Stylelint 16.0.0. See the [migration guide to v16](https://stylelint.io/migration-guide/to-16#removed-deprecated-stylistic-rules).
Please remove the rules from your Stylelint configuration if you want to continue to disabling the rules.

**Comment by openmindculture** [comment]
Thanks for clarifying, I understand and I removed the rules but that should not be necessary to prevent breaking builds after upgrading. Ignoring unknown configuration lines would make stylelint more robust, like CSS ignores unknown rules to make it backwards compatible.
